docs/baseline.md

```diff
@@ -60,7 +60,7 @@ Baseline status cannot or will not satisfy the following non-goals:
   If a web developer needs to support a globally uncommon or discontinued browser (e.g., Internet Explorer 11), then the developer needs to know the specific limitations of that browser, not a broad overview of developers’ most commonly required browsers.
   Baseline can’t be both.
 * **Identify support in assistive technology.**
-  Baseline does not cover support in assistive technology (AT), which includes screen readers, screen magnifiers, voice control, and so on.
+  Baseline does not cover support for screen readers, screen magnifiers, voice control, and other assistive technology that browsers do not expose to developers.
```

Comment:
I might clarify it like this:

```suggestion
  Baseline does not cover support for screen readers, screen magnifiers, voice control, and other assistive technology that is not built into browsers.
```

I would also make it "Identify support in assistive technology outside of browsers" above.

Reply:
+1 to the suggested change.